src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -279,6 +279,15 @@ public static class CLIOptions
             JsonKey = "test-projects"
         };
 
+        public static readonly CLIOption<string> MutationLevel = new CLIOption<string>
```

Comment:
I would prefer something like Mode or MutationMode

Mode > MutationMode tbh

Reply:
I disagree. Level suggests there are gradations and each level adds more mutatants. Mode says more about how mutations should be placed or something. Not about how many mutations are placed. 

I'd like to think about this like loglevel. The lower the loglevel the more logs are kept. In stryker the higher the mutationlevel the more mutations are being placed.

Reply:
@Mobrockers I've heard no one else say they prefer Mode over Level so I'd say we go with Level. I'd like to merge this today so we can add complex mutators during hacktoberfest :)